src/nvapi_d3d.cpp

```diff
@@ -272,7 +272,16 @@ extern "C" {
         if (nvapiD3dInstance->IsUsingLfx() || !NvapiD3dLowLatencyDevice::SupportsLowLatency(pDev))
             return NoImplementation(n, alreadyLoggedNoImpl);
 
-        if (!NvapiD3dLowLatencyDevice::SetLatencyMarker(pDev, pSetLatencyMarkerParams->frameID, pSetLatencyMarkerParams->markerType))
+        auto markerType = NvapiD3dLowLatencyDevice::ToMarkerType(pSetLatencyMarkerParams->markerType);
+        if (!markerType.has_value()) {
+            // Silently drop unsupported marker types
+            if (!alreadyLoggedOk)
+                log::info(str::format("Not supported NV_LATENCY_MARKER_TYPE: ", pSetLatencyMarkerParams->markerType));
```

Comment:
This can probably be removed now (and from the other function as well), especially since it's unlikely to ever log anything when first valid call with correct marker type will set `alreadyLoggedOk`.

Reply:
thx, changed with new `bool` to log this at least once.

Reply:
Yup, that should work fine.